dashboard/test/ui/features/step_definitions/steps.rb

```diff
@@ -509,8 +509,9 @@ def select_dropdown(element, option_text, load)
 end
 
 When /^I click selector "([^"]*)" once I see it(?: to load a new (page|tab))?$/ do |selector, load|
+  wait_for_jquery
   wait_until do
-    @browser.execute_script("return $(\"#{selector}:visible\").length != 0;")
+    @browser.execute_script(jquery_is_element_visible(selector))
```

Comment:
from https://api.jquery.com/visible-selector/ it looks like the `:visible` selector behaves slightly differently than this method does:

> Elements with `visibility: hidden` or `opacity: 0` are considered visible, since they still consume space in the layout.

I'm not sure if this will come up, but it could be worth finding a way to test in all browsers on the test machine before merging, or else merge at a time we are not on the critical path for the daily deploy.

Reply:
Thanks for pointing this out!  I'm going to merge it tonight and will keep an eye on it.